addons/binding/org.openhab.binding.omnilink/ESH-INF/thing/thermostat.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="omnilink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+    <thing-type id="thermostat">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="controller" />
+        </supported-bridge-type-refs>
+        <label>Thermostat</label>
+        <description>Thermostat</description>
+        <channels>
+            <channel id="status" typeId="thermostat_status" />
+            <channel id="temperatureC" typeId="thermostat_temperature" />
```

Comment:
I totally forgot that there is a  user preference for C vs F is stored as a controller property, so I can remove the C and F variations